web/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.6-alpine3.7
+FROM alpine:latest
```

Comment:
Why do we need to switch to `alpine` base image instead of using the one we are currently using?

Reply:
python:* packages build python from source and into /usr/local. That makes it harder to reuse alpine packages that interface with python (eg. xapian's python bindings that haystack then uses).
By going for "pure" alpine and its packages, we have a single source of binaries.

Reply:
It sounds reasonable, but let's not use the `latest` tag, I'd much prefer that we pick whatever is the latest version (3.7?).